tests/plugins/colordialog/manual/addhash.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: tc, feature, colordialog, 565, 4.8.0
```

Comment:
As it was discussed some time ago there should either `bug` or `feature` tag? Or does it change since then?